### PR TITLE
Defi scam article

### PR DESCRIPTION
**Pull request by CryptoManion**

### CODE REVIEW

File: docs/defi/DeFi-scams.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+# Basic Security Rules
+
+These basic rules to protect yourself can apply to all Web 1 to Web 3 and would still be available up to Web 10. 
+ 
+- **Don’t trust anyone**; always verify the info by yourself. Check the official sources for built-in applications, software, wallets, dapps, or smart contracts;
+- Protecting yourself means protecting **your online identity**. We recommend you look at this article on the Horizen Academy to learn more about [protecting your digital privacy](https://www.horizen.io/academy/digital-privacy/) (2FA, browsers, virtual phone numbers, etc.);
+- **Use multiple MetaMask/web wallet accounts** connected to your browsers. If you want to participate in airdrops/tests, do it from a wallet with few cryptocurrencies. You can use them on different browsers and even different computers. One browser/computer will be reserved for more serious activities and the other for testing. It is the same as credit card payments on the Internet, for these purchases, it is recommended to use single-use cards, available in many neo-banks;
+- **Keep your private keys in a safe place**, do not copy/save them on your computer, and **never share them with anyone**;
```
Comment (yurikoinaba) [inline docs/defi/DeFi-scams.md:30]:
```suggestion
- **Keep your private keys in a safe place**, do not copy/save them on your computer, and **never share them with anyone**.
```

File: docs/defi/DeFi-scams.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+- How to Do Your Own Research (DYOR) not to get scammed.
+
+We recommend you read the Horizen Academy article about the private key, which will bring you lots of information about blockchain and crypto technologies, new articles are added often, so feel free to bookmark them and come back! 
+
+If you are here to learn about how to do your research and some tips to not get scammed, this article is for you. **Note that new scamming schemes can appear and that this list is not exhaustive**.
+# Basic Security Rules
+
+These basic rules to protect yourself can apply to all Web 1 to Web 3 and would still be available up to Web 10. 
+ 
+- **Don’t trust anyone**; always verify the info by yourself. Check the official sources for built-in applications, software, wallets, dapps, or smart contracts;
```
[the remark this answers is not in the document]
Comment (yurikoinaba) [reply docs/defi/DeFi-scams.md:27]: End all bullet points w a period instead of semicolon

File: docs/defi/DeFi-scams.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+## DeFi and liquidity pool rug pulls
+Rug pulls consist of withdrawing all the funds users place on a project that looks promising, often through a backdoor written in the code.
+
+Example: [SQUID token](https://www.bbc.com/news/business-59129466), [Luna](https://coingeek.com/solana-sees-first-rug-pull-luna-yield-disappears-with-6-7m-in-digital-currency/).
+
+About **liquidity pool** rug pulls, the scammer uses new DeFi products with new coins to make the user invest and steal his money. 
+
+_*In a liquidity pool, users make cryptocurrencies available, on a pair at a time, for other users to borrow, lend, or trade via the pool. Exchanges are facilitated by moving from an order book technology, where a buyer must necessarily have a seller in front of him, to an AMM, Automated Market Maker, where the seller or buyer is no longer required in front of him to conclude the exchange._
+
+In these rug puls, users are incentivized to provide liquidity on a pair, for which they usually have to exchange (here, we say “swap”) BTC for the newly created token. To conclude the scam, the malicious actors remove all BTC from the pool.
```
Comment (yurikoinaba) [inline docs/defi/DeFi-scams.md:85]:
```suggestion
In these rug pulls, users are incentivized to provide liquidity on a pair, for which they usually have to exchange (here, we say “swap”) BTC for the newly created token. To conclude the scam, the malicious actors remove all BTC from the pool.
```

**Review by yurikoinaba — COMMENTED** [review]
Looks good from a technical standpoint, and left some comments on the text.

It would be good for someone on the marketing team (maybe Bryan) to help proofread.

File: docs/defi/DeFi-scams.md [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+slug: /defi-scams/
+description: How to protect yourself in the Web worlds, a focus on DeFi and Web3
+image: https://www.horizen.io/academy/img/defi-scams/Horizen_Academy-meta-image-DEFI-scams.jpg
+last_update:
+  date: 1/14/2024
+  author: Horizen Contributor
+
+---
+# Ode to DYOR
+One of the most important revolutions brought with blockchain and cryptocurrencies is ownership and banking freedom. But with great power comes responsibilities. This new era comes with education, the new paradigm shift; not knowing and relying totally on third parties means losing security and ownership and is not the goal of these new technologies.
```
[the remark this answers is not in the document]
Comment (CryptoManion) [reply docs/defi/DeFi-scams.md:14]: oh yes, indeed :) thanks for flagging

[the remark this answers is not in the document]
Comment (CryptoManion) [reply docs/defi/DeFi-scams.md:14]: Can you suggest the change? i am so lost again on Github :')

Comment (jmontesdeocanuez) [inline docs/defi/DeFi-scams.md:14]:
```suggestion
One of the most important revolutions brought with blockchain and cryptocurrencies is ownership and banking freedom. But with great power comes great responsibility. This new era comes with education, the new paradigm shift; not knowing and relying totally on third parties means losing security and ownership and is not the goal of these new technologies.
```

**Review by yurikoinaba — APPROVED** [review]

**Comment by CryptoManion** [comment]
The meta image is not displaying here
Reviewed by Bryan it's good to go

File: docs/defi/DeFi-scams.md [hunk 5]
```diff
@@ -0,0 +1,203 @@
+---
+
+sidebar_label: DeFi scams
+title: What are DeFi scams and how to protect yourself
+slug: /defi-scams/
+description: How to protect yourself in the digital age, a focus on DeFi and Web3
+image: https://www.horizen.io/academy/img/Horizen_Academy-meta-image-DEFI-scams.jpg
```
Comment (yurikoinaba) [inline docs/defi/DeFi-scams.md:7]:
```suggestion
image: https://www.horizen.io/academy/img/defi-scams/Horizen_Academy-meta-image-DEFI-scams.jpg
```

**Review by yurikoinaba — APPROVED** [review]

**Pull request merged**